src/Compilers/Core/Portable/Diagnostic/LocalizableResourceString.cs

```diff
@@ -21,11 +21,6 @@ public sealed class LocalizableResourceString : LocalizableString, IObjectWritab
         private readonly Type _resourceSource;
         private readonly string[] _formatArguments;
 
-        static LocalizableResourceString()
-        {
-            ObjectBinder.RegisterTypeReader(typeof(LocalizableResourceString), reader => new LocalizableResourceString(reader));
-        }
```

Comment:
Need an explanation on why this was removed.

Reply:
This is fundamentally untrimmable, but the only usage I found was in tests.

Reply:
Are we sure this isn't used for diagnostics being serialized across processes?

Reply:
I tried and couldn't find a place where it was used. The integration tests are also passing. I'm not sure how else to verify this.